index.html

```diff
@@ -55,23 +61,90 @@
       </div>
       
 
-      <nav class="white">
+       <!--Navbar-->  
+        <!-- <nav class="white" style="font-family: 'Montserrat', sans-serif;">
           <div class="nav-wrapper">
-            <a href="index.html" class="brand-logo black-text">Doc2Pen</a>
+            <a href="index.html" class="brand-logo black-text">Doc<span class="blue-text">2</span>Pen</a>
```

Comment:
Do not change NavBar style for now. We will deal with it once all three pages are complete